### ISSUE TITLE
When an activity preview includes both links and images, links previews should not be skipped 

### ISSUE THREAD

**Issue by ivanminutillo** [bonfire-networks/bonfire-app#494]
or alternatively be added as a separate list below the images compositions ?

**Comment by mayel** [comment]
Both are shown on my end:
<img width="663" alt="Screenshot 2023-03-20 at 08 43 07" src="https://user-images.githubusercontent.com/115318/226204885-85bfd47f-b154-4360-8164-e33fccf991af.png">

**Comment by ivanminutillo** [comment]
yeah the issue title is misleading. Links are currently visualised within the images gallery which is confusing, instead they should be visualised separated from the images - whe both are present

**Comment by ivanminutillo** [comment]
Done!

![image](https://user-images.githubusercontent.com/1852065/229160402-2878b522-2c09-40c3-9976-6ac00d7b2cb9.png)
